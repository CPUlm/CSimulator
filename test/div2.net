INPUT
OUTPUT u, v, w, x
VAR
	a,b,c,qqch,uInit, vInit, wInit, xInit, u,v,w,x
IN
a = REG u
b = REG v
c = REG w
qqch = OR b c
uInit = ROM 1 1 0
vInit = ROM 1 1 0
wInit = ROM 1 1 0
xInit = ROM 1 1 0
u = MUX qqch uInit 0
v = MUX qqch vInit a
w = MUX qqch wInit b
x = MUX qqch xInit c
