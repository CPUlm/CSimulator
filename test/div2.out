Step 1:
=> u = 1
=> v = 0
=> w = 1
=> x = 1
Step 2:
=> u = 0
=> v = 1
=> w = 0
=> x = 1
Step 3:
=> u = 0
=> v = 0
=> w = 1
=> x = 0
Step 4:
=> u = 0
=> v = 0
=> w = 0
=> x = 1
Step 5:
=> u = 1
=> v = 0
=> w = 1
=> x = 1
Step 6:
=> u = 0
=> v = 1
=> w = 0
=> x = 1
Step 7:
=> u = 0
=> v = 0
=> w = 1
=> x = 0
Step 8:
=> u = 0
=> v = 0
=> w = 0
=> x = 1
Step 9:
=> u = 1
=> v = 0
=> w = 1
=> x = 1
Step 10:
=> u = 0
=> v = 1
=> w = 0
=> x = 1
